README.md

```diff
@@ -36,7 +36,10 @@ $ npm install --save-dev eslint eslint-plugin-n
         "n/prefer-global/url": ["error", "always"],
         "n/prefer-promises/dns": "error",
         "n/prefer-promises/fs": "error"
-    }
+    },
+    "plugins": [
+        "n"
+    ]
```

Comment:
it's not needed any more - it has been set in the recommended preset:
https://github.com/weiran-zsd/eslint-plugin-node/blob/a6a31bbd4fcf4bceff11d383f851cd9d9b7ce05b/lib/configs/recommended-script.js#L19